api/pom.xml

```diff
@@ -105,7 +105,7 @@
 		<dependency>
 			<groupId>org.openmrs.module</groupId>
 			<artifactId>reportingcompatibility-api</artifactId>
-			<version>2.0.3</version>
+			<version>2.0.6</version>
```

Comment:
@dkayiwa making this change (since its what the distribution package uses) causes the error 

[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.1:compile (default-compile) on project ugandaemrreports-api: Compilation failure
[ERROR] openmrs-module-ugandaemr-reports/api/src/main/java/org/openmrs/module/ugandaemrreports/reporting/cohort/evaluator/CalculationCohortDefinitionEvaluator.java:[82,88] incompatible types: org.openmrs.cohort.Cohort cannot be converted to org.openmrs.Cohort

Reply:
What was the problem with version 2.0.3?